File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        
        int length = nums.size();

        std::vector<int> ans;

        ans.resize(2 * length);

        for (int i = 0; i < length; i++) {
            ans[i] = nums[i];
            ans[i+(length)] = nums[i];
        }

        return ans;
    }
};

File: Data Structures & Algorithms/binary-tree-right-side-view/submission-0.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def rightSideView(self, root: Optional[TreeNode]) -> List[int]:
        
        if not root:
            return []

        queue = deque([root])

        ans = []

        node = root

        while queue:

            lvl = []

            lvlsize = len(queue)

            for n in range(lvlsize):

                node = queue.popleft()

                if node.left:
                    queue.append(node.left)
                if node.right:
                    queue.append(node.right)

                lvl.append(node.val)
            
            ans.append(lvl.pop())
        
        return ans
            

